src/chapters/adhoc-connections.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<chapter xml:id="adhoc-connections" xmlns="http://docbook.org/ns/docbook" version="5.0" xml:lang="en"
+    xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Ad-hoc Connections</title>
+    <indexterm>
+        <primary>adhoc-connections</primary>
+        <secondary>adhoc</secondary>
+        <secondary>quickconnect</secondary>
+    </indexterm>
+    <para>The quickconnect extension provides a connection bar on the Guacamole Client home page
+        that allows users to type in the URI of a server to which they want to connect and the client
+        will parse the URI and immediately establish the connection.  The purpose of the extension is
+        to allow situations where administrators want to allow users the flexibility of establishing
+        their own connections without having to grant them access to edit connections or even to have
+        to create the connections at all, aside from typing the URI.</para>
+    <important>
+        <para>There are several implications of using this extension that should be well-understood
+            by administrators prior to implementing it:</para>
+        <itemizedlist>
+            <listitem><para>Connections established with this extension are created in-memory
+                    and only persist until the user logs out or the browser is closed.</para></listitem>
```

Comment:
> ... or the browser is closed.

Is this the case? I thought the connections persist in server-side memory, within the user's session, independently of whether the browser has been closed.

Reply:
Modified that a little.